types/result.go

```diff
@@ -57,7 +57,7 @@ func (r TransactionResult) Reverted() bool {
 // TransactionResultDebug provides details about unsuccessful transaction execution
 type TransactionResultDebug struct {
 	Message string
-	Meta    map[string]string
+	Meta    map[string]any
```

Comment:
does this have to be any?

Reply:
@janezpodhostnik to use it directly with logger, I think it has to be any ( otherwise logger is skipping somehow )